mqttclients/c/mosquitto_client_extensions/json_handlers/geo_json_handler.h

```diff
@@ -0,0 +1,93 @@
+/* Copyright (c) Microsoft Corporation. All rights reserved. */
+/* SPDX-License-Identifier: MIT */
+
+#ifndef GEO_JSON_HANDLER_H
+#define GEO_JSON_HANDLER_H
+
+#include "mosquitto.h"
+#include <json-c/json.h>
+
+typedef struct mosquitto_payload
+{
+  char* payload;
+  size_t payload_length;
+  size_t max_payload_length;
+} mosquitto_payload;
+
+// Reference generic geojson struct if another type is needed in the future
```

Comment:
Could we get a more descriptive comment here if this commented code is to be kept? Otherwise it should be removed.

Reply:
For the purposes of this sample, we aren't planning on showing any other types of geojson, so I went ahead and removed this to keep from being confusing.